beancount-stubs/core/realization.pyi

```diff
@@ -40,7 +40,7 @@ def realize(
 ) -> RealAccount: ...
 def postings_by_account(
     entries: Iterable[Directive],
-) -> list[dict[Account, TxnPosting | Directive]]: ...
+) -> dict[Account, list[Account, TxnPosting | Directive]]: ...
```

Comment:
This should be just `TxnPosting | Directive`, no?

Reply:
D'oh.